src/core/catalog.js

```diff
@@ -334,7 +334,7 @@ class Catalog {
         docBaseUrl: this.baseUrl,
         docAttachments: this.attachments,
       });
-      const title = outlineDict.get("Title");
+      const title = outlineDict.get("Title") || "";
```

Comment:
I'd suggest reverting this line, and *instead* change https://github.com/mozilla/pdf.js/blob/55db43966eaf99b1292ae913fe2ae3eae7d0e8ad/src/core/catalog.js#L360 into the following to *properly* validate the data:
```javascript
title: typeof title === "string" ? stringToPDFString(title) : "", 
```